### PR TITLE
Swiping Resolved

### PR DESCRIPTION
**Pull request by sid-1207**
Fixed Calendar Swiping

### CODE REVIEW

File: lib/screens/home_page.dart [hunk 1]
```diff
@@ -78,8 +77,8 @@ class HomePage extends StatefulWidget {
 class _HomePageState extends State<HomePage> {
   int month;
   int year;
-  int index = 0;
   bool _loading = false;
+  int prevIndex = 999, currentIndex = 1000;
```
Comment (hashirshoaeb) [inline lib/screens/home_page.dart:81]: feels more like 'jugaad' haha

File: lib/screens/home_page.dart [hunk 2]
```diff
@@ -134,16 +133,29 @@ class _HomePageState extends State<HomePage> {
...
-                onHorizontalDragEnd: onHorizontalDragEnd,
-                child: Container(
-                  padding: c.EdgeInsets.symmetric(horizontal: 12),
-                  child: Month(
-                    month: month,
-                    year: year,
-                  ),
-                ),
-              ),
+              child: c.PageView.builder(
```
Comment (hashirshoaeb) [inline lib/screens/home_page.dart:136]: Nice work

File: pubspec.yaml [hunk 3]
```diff
@@ -53,6 +53,8 @@ dev_dependencies:
   # Local database
   hive_generator: ^0.8.1
 
+  swipedetector: 1.2.0
```
Comment (hashirshoaeb) [inline pubspec.yaml:56]: Need this to me removed, since we are not using it.

**Review by hashirshoaeb — APPROVED** [review]

**Comment by hashirshoaeb** [comment]
I'm hesitating to merge 'jugad' tho. 😅  Really appreciate your effort. Thanks for that. 🔥

**Comment by sid-1207** [comment]
> I'm hesitating to merge 'jugad' tho. 😅  Really appreciate your effort. Thanks for that. 🔥

🥲
It was great working with you
Thanks for the help and support

**Comment by hashirshoaeb** [comment]
My inbox is always open 😉

**Pull request closed without merging**